### PR TITLE
docs(README): add deployment section

### PR DESCRIPTION
**Pull request by mikehardy**
Documenting deployment process after first successful (I think) deployment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,4 +20,24 @@ We are constantly working to improve this project, help us with your donation as
...
+
+### Development
+
+PRs should be made against and merged into the [`dev-5.x`](https://github.com/hossein-zare/react-native-dropdown-picker) branch, which is set as the `default` branch on github.
+
+### Release
+
+Releases are currently made from teh [`5.x`](https://github.com/hossein-zare/react-native-dropdown-picker/tree/5.x) branch.
+
+To make a new release, follow these steps:
```
Comment (mikehardy) [inline README.md:39]:
```suggestion
## Release Process

To make a new release, follow these steps:
```

File: README.md [hunk 2]
```diff
@@ -20,4 +20,24 @@ We are constantly working to improve this project, help us with your donation as
...
+
+* Verify the `dev-5.x` branch has all the changes desired in a release and works well
+* Make and merge a final PR into `dev-5.x` that increments the version number in `package.json`
+* Make and merge a PR from the `dev-5.x` branch to the `5.x` branch
+* Using the GitHub web UI, draft a new release using tag name `v5.x.x` (replace the `x` values as appropriate of course), against target `5.x`, with release name `v5.x.x` (again, with appropriate numbers in place of `x` of course)
```
Comment (mikehardy) [inline README.md]:
```suggestion
* Verify the development branch has all the changes desired in a release and works well
* Make and merge a final PR into development branch that increments the version number in `package.json`
* Make and merge a PR from the development branch to the release branch
* Using the GitHub web UI, draft a new release using tag name `vx.x.x` (replace the `x` values as appropriate of course), with the release branch as the target, with release name `vx.x.x` (again, with appropriate numbers in place of `x` of course)
```

**Comment by mikehardy** [comment]
Looks good to me now, would love a quick scan @hossein-zare if you have a moment

image

File: README.md [hunk 3]
```diff
@@ -20,4 +20,26 @@ We are constantly working to improve this project, help us with your donation as
...
+
+## Branches in use
+
+### Development
+
+PRs should be made against and merged into the [`dev-5.x`](https://github.com/hossein-zare/react-native-dropdown-picker) branch, which is set as the `default` branch on github.
+
+### Release
+
+Releases are currently made from teh [`5.x`](https://github.com/hossein-zare/react-native-dropdown-picker/tree/5.x) branch.
```
Comment (hossein-zare) [inline README.md]: There's a typo "teh" :)

**Comment by hossein-zare** [comment]
I invited you as a maintainer to NPM.
https://www.npmjs.com/package/react-native-dropdown-picker

File: README.md (hunk 3 above)
Comment (mikehardy) [reply README.md]: :facepalm: and that's why review never ever hurts :-). Great spot

Comment (mikehardy) [inline README.md]:
```suggestion
Releases are currently made from the [`5.x`](https://github.com/hossein-zare/react-native-dropdown-picker/tree/5.x) branch.
```

**Review by hossein-zare — APPROVED** [review]
Cool, Thank you.

**Pull request merged**